statefun-sdk/src/main/java/org/apache/flink/statefun/sdk/state/PersistedAppendingBuffer.java

```diff
@@ -160,6 +160,13 @@ public void clear() {
     accessor.clear();
   }
 
+  @Override
```

Comment:
I think that it would be somewhat not intuitive for users not to see the actual value, what do you think?